benchmarks/benchmark_morphology.py

```diff
@@ -36,3 +37,14 @@ def peakmem_reference(self, *args):
     def peakmem_watershed(self, seed_count, connectivity, compactness):
         morphology.watershed(self.image, seed_count, connectivity,
                              compactness=compactness)
+
+
+class WatershedParallel(object):
+
+    def setup(self):
+        image = filters.sobel(data.coins())
+        self.images = ((image, 100) for _ in range(4))
+
+    def time_watershed_parallel(self):
+        with ThreadPool(4) as pool:
```

Comment:
Initially I created the ThreadPool inside the setup method so that this initialization wouldn't taint the benchmark. However the benchmark would always fail when using `asv run` with `ValueError: Pool not running` despite passing with with `asv dev`. Moving the initialization into the `time_`-function seems to work now. I figure asv does some threading behind the scenes which interfered with the ThreadPool?